pkg/cri/opts/spec_windows_opts.go

```diff
@@ -1,3 +1,6 @@
+//go:build windows
```

Comment:
Do we need this build constraints here?
We should be able to generate OCI cross platform.
See https://github.com/containerd/containerd/pull/7928 for context.

Reply:
I had tried to avoid adding the windows specific builds before the last update as well but the problem is with importing windows pkg ( the use of windows.EscapeArg() call in escapeAndCombineArgsWindows() ). 
There seems to be a restriction :

package github.com/containerd/containerd/cmd/containerd
        imports github.com/containerd/containerd/cmd/containerd/builtins
        imports github.com/containerd/containerd/pkg/cri
        imports github.com/containerd/containerd/pkg/cri/sbserver
        imports github.com/containerd/containerd/pkg/cri/opts
        **imports golang.org/x/sys/windows: build constraints exclude all Go files in C:\Users\kiashok\go\src\github.com\containerd\upstreamCtrd\argsEscaped\containerd\vendor\golang.org\x\sys\windows**

In the example Danny pointed out too, there is a build restriction like "//go:build !linux && !windows"

Reply:
ping on this :)

Reply:
If it's platform dependent, then yes, we should move it to a `_windows.go` file.

Reply:
Alright, will move windows specific functions from this file to pkg/cri/opts/spec_windows.go